docs/ckecli.md

```diff
@@ -299,6 +300,11 @@ Cancel the specified reboot queue entry.
 
 Cancel all the reboot queue entries.
 
+### `ckecli reboot-queue reset-backoff`
+
+Reset `drain_backoff_count` and `drain_backoff_expire` of the entries in reboot queue.
+Resetting these values allows CKE to try draining nodes again immediately.
```

Comment:
- It is better to use a verb that guarantees the action being actually done, instead of using _allow_.
- This command cause the retry of entire reboot sequence, not only draining.
- Try _V-ing_ means to check the result of the action. Use "try to" if it may fail.

```suggestion
Resetting these values makes CKE try to reboot nodes again immediately.
```

Reply:
Thank you. I fixed it.